node/services/config.go

```diff
@@ -17,10 +17,13 @@ type Config struct {
 	// TrustedHash is the Block/Header hash that Nodes use as starting point for header synchronization.
 	// Only affects the node once on initial sync.
 	TrustedHash string
-	// TrustedPeer is the peer we trust to fetch headers from.
-	// Note: The trusted does *not* imply Headers are not verified, but trusted as reliable to fetch headers
-	// at any moment.
+	// TrustedPeer is the multiaddr of the peer we trust to fetch headers from.
+	// Note: The trusted does *not* imply Headers are not verified, but trusted
+	// as reliable to fetch headers at any moment.
 	TrustedPeer string
+
+	// TODO @renaynay: document
```

Comment:
Can we document this now?

Reply:
Documented in [b3b5707](https://github.com/celestiaorg/celestia-node/pull/420/commits/b3b5707466aa31385127857142f5ad8b834bcb1d)